src/main/java/net/bramp/commons/lang3/math/gson/FractionAdapter.java

```diff
@@ -50,12 +50,14 @@ public Fraction read(JsonReader reader) throws IOException {
     String fraction = reader.nextString().trim();
 
     // Ambiguous as to what 0/0 is, but FFmpeg seems to think it's zero
-    if (zeroByZero != null && fraction.equals("0/0"))
+    if (zeroByZero != null && "0/0".equals(fraction)) {
```

Comment:
It's a common Java practice to [always put the string literal first](https://en.wikibooks.org/wiki/Java_Programming/Preventing_NullPointerException#Comparing_string_variable_with_a_string_literal) to avoid potential`NullPointerException`.